app/components/view-epic-list/macro.test.js

```diff
@@ -0,0 +1,272 @@
+import request from 'supertest';
+import cheerio from 'cheerio';
+import { createTestHarness } from '../../testUtils/testHarness';
+
+const macroWrapper = `{% from './components/view-epic-list/macro.njk' import viewEpicList %}
+                        {{ viewEpicList(params) }}`;
+
+describe('view-epic-list', () => {
+  it('should render the met data when provided', (done) => {
+    const context = {
+      params: {
+        dataTestId: 'some-data-identifier',
+        data: [
+          {
+            id: 'C14E1',
+            name: 'access prescribable items',
+          },
+          {
+            id: 'C14E2',
+            name: 'manage Formularies',
+          },
+          {
+            id: 'C14E13',
+            name: 'access Patient Record',
+          },
+        ],
+        type: 'met',
+      },
+    };
+
+    const dummyApp = createTestHarness(macroWrapper, context);
+    request(dummyApp)
+      .get('/')
+      .then((res) => {
+        const $ = cheerio.load(res.text);
+        expect($('[data-test-id="some-data-identifier"] ul').length).toEqual(1);
+        expect($('[data-test-id="some-data-identifier"] li').length).toEqual(3);
+
+        done();
+      });
+  });
+
+  it('should not render the met data when not provided', (done) => {
+    const context = {
+      params: {
+        dataTestId: 'some-data-identifier',
+        data: [],
+        type: 'met',
+      },
+    };
+
+    const dummyApp = createTestHarness(macroWrapper, context);
+    request(dummyApp)
+      .get('/')
+      .then((res) => {
+        const $ = cheerio.load(res.text);
+        expect($('[data-test-id="some-data-identifier"]').length).toEqual(0);
+
+        done();
+      });
+  });
+
+  it('should render the not-met data when provided', (done) => {
+    const context = {
+      params: {
+        dataTestId: 'some-data-identifier',
+        data: [
+          {
+            id: 'C14E1',
+            name: 'access prescribable items',
+          },
+          {
+            id: 'C14E2',
+            name: 'manage Formularies',
+          },
+          {
+            id: 'C14E13',
+            name: 'access Patient Record',
+          },
+        ],
+        type: 'not-met',
+      },
+    };
+
+    const dummyApp = createTestHarness(macroWrapper, context);
+    request(dummyApp)
+      .get('/')
+      .then((res) => {
+        const $ = cheerio.load(res.text);
+        expect($('[data-test-id="some-data-identifier"] ul').length).toEqual(1);
+        expect($('[data-test-id="some-data-identifier"] li').length).toEqual(3);
```

Comment:
Same as with the met comment above. Check the name and ids as well as the svgIcon